lib/active_merchant/billing/gateways/rapyd.rb

```diff
@@ -74,10 +77,12 @@ def store(payment, options = {})
         add_payment_fields(post, options)
         add_payment_urls(post, options, 'store')
         add_address(post, payment, options)
+        add_idempotency(options)
         commit(:post, 'customers', post)
       end
 
       def unstore(customer)
+        add_idempotency(options)
```

Comment:
I don't think `options` exists in this context.

Reply:
Right, why didn't the unit tests catch that. Let me look at that.

Reply:
Fix it